index.html

```diff
@@ -3,24 +3,25 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Some Student</title>
+  <title>Dustin Walker</title>
 
   <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
+  <link rel="stylesheet" href="styles/style.css">
 </head>
 
 <body>
 
-<header class="z">
+<header class="hero__container">
   <nav class="top-nav">
     <div class="z__container">
-      <div class="foo-flex">
-        <h1 class="foo-bar"><a href="#">Some Student</a></h1>
+      <div class="hero">
+        <h1 class="nav-group a"><a href="#">Dustin Walker</a></h1>
 
-        <ul class="foo-bar"
-          <li class="foo-bar__x"><a href="#">Home</a></li>
-          <li class="foo-bar__x"><a href="#">About</a></li>
-          <li class="foo-bar__x"><a href="#">Contact</a></li>
+        <ul class="ul.nav-group">
```

Comment:
When looking at the CSS selector `ul.nav-group` this means that the browser is looking for an element that is a `ul` AND has a class of `nav-group`. One way to remember this is that class names will not have `.` in them.